src/main/java/com/thealgorithms/others/RotateMatrixBy90Degrees.java

```diff
@@ -6,8 +6,8 @@
  */
 import java.util.Scanner;
 
-final class Rotate_by_90_degrees {
-    private Rotate_by_90_degrees() {
+final class RotateMatrixBy90Degrees  {
```

Comment:
This should make `clang-format` happy:

```suggestion
final class RotateMatrixBy90Degrees {
```

Reply:
> This should make `clang-format` happy:

Thank you for your perspective! 😄 
Since this is my second attempt at contributing, I'm not very familiar with how to properly check. I did some testing before submitting, but it wasn't comprehensive. I will improve moving forward.

Reply:
In order to _clang-format_ all of the java files, you can run something like:
```shell
find . -regex '.*\.java' -exec clang-format -style=file -i {} \;
```